### ISSUE TITLE
boat extraction fails

### ISSUE THREAD

**Issue by ScruffyAT** [CaptainPStar/co10_Escape#161]
Fails with "left somebody behind" ending, maybe because of sitting in escort?
Script accounts for this, should be same with land extraction

**Comment by aussie-battler** [comment]
Some players believe it happens if you get into the first boat with no lights on. Maybe it is the escort boat?

I might see if there is a gun boat that you cant hop in & use that as the first boat.

**Comment by ScruffyAT** [comment]
Yes, that's what I think, too. But the script checks for that and I tested that when implementing. Hasn't happened with the car extraction for us (yet?), and that's mostly a copy/paste. Other suspicion is something going on with turrets/FFV positions.
If you find something, please tell :)

**Comment by aussie-battler** [comment]
Not sure if this is related, has a car extract that didnt move. We just sat in the vehicle & it didn't move. After a few minutes we got mission successful.  It was on Western Sahara DLC getting into a APC (which is set to both extraction vehicle & escort).

a3e_arr_extraction_car = [
	"B_D_Truck_01_transport_lxWS"	//17
	,"B_D_Truck_01_covered_lxWS"	//17
	,"B_D_APC_Tracked_01_rcws_lxWS"	//8
	,"B_D_APC_Wheeled_01_cannon_lxWS"	//8
	,"B_D_APC_Wheeled_01_command_lxWS"];	//8
															
   
a3e_arr_extraction_car_escort = [
	"B_D_APC_Tracked_01_aa_lxWS",
	"B_D_APC_Wheeled_01_atgm_lxWS",
	"B_D_MBT_01_cannon_lxWS",
	"B_D_MBT_01_TUSK_lxWS"];

**Comment by CaptainPStar** [comment]
All extractions except heli seem to have a few problems.
